advanced/Scripts/list.sh

```diff
@@ -106,7 +115,7 @@ ProcessDomainList() {
         # if delmode then remove from desired list but do not add to the other
         if ${addmode}; then
             AddDomain "${dom}" "${listType}"
-            if [[ ! "${listType}" == "regex_blacklist" ]]; then
+            if [[ ! "${listType}" == "regex_"*"list" ]]; then
```

Comment:
Checking if the list type is of a regex list is done differently here and above. Only one way should be used, and probably should be made into a function.